src/main/java/column/store/api/column/IdColumn.java

```diff
@@ -0,0 +1,7 @@
+package column.store.api.column;
+
+public final class IdColumn extends BaseColumn {
```

Comment:
I assume this is only used for explicitly identifying related column values ?

Reply:
A column for ids, i.e. user-ids, etc. Values which are inherently random, and therefore compress bad.